Client/qtTeamTalk/mainwindow.h

```diff
@@ -154,6 +154,7 @@ class MainWindow : public QMainWindow
 
     //current host
     HostEntry m_host;
+    bool m_latesthost = false;

```

Comment:
shouldn't we put 'm_latesthost' into the HostEntry-struct? It could also simply other functions (like setServerEntry, deleteServerEntry)

Reply:
I didn't see a real utility to put this in HostEntry struct, we already have all we need to compare latesthost and saved host in common.cpp. Also if we add it to HostEntry struct we probably have to store it in config file, so update config at first startup of this version. And, I'm not sure it's really useful to store this in config as we already have different sections for latest host and server entries.
So, as you want :), I can move it to HostEntry struct if you find it useful, but I think we can keep as currently.

Reply:
ok